### DISCUSSION TITLE
Pointer Capture

### DISCUSSION THREAD

**Discussion by JyanJohn** [ppy/osu#15654] [Ideas] [answered]
Hello, is it possible for the osu lazer to capture my mouse. Im connecting my mouse and keyboard to my phone. My expected behaviour is using the game cursor instead of the android cursor

**Comment by peppy** [comment]
Last time we checked there was no API for this. Are you seeing other games that do this correctly?

**Comment by frenzibyte** [comment, marked as answer]
This is tracked in https://github.com/ppy/osu-framework/issues/4373, and there is now a way in API level 26 (i.e. 8.0+), if your Android phone is running at lower than 8.0 then there is no way, nope.

**Reply by JyanJohn** [reply]
Im in Android 11 (API 30)

**Reply by frenzibyte** [reply]
Then wait until https://github.com/ppy/osu-framework/issues/4373 gets resolved.
